### ISSUE TITLE
Let users message each other

### ISSUE THREAD

**Issue by Appshure** [akirk/friends#98]
I create a support topic in wordpress.org but haven't gotten a response. Is there a way to let friends message each other?

**Comment by akirk** [comment]
Sorry, I missed the message there. Yes, there is a feature called "Blog to blog messaging", see the demo video on #84.

**Comment by Appshure** [comment]
Thanks.
Seems like the feature only works with Gutenberg, my site runs multisite and I want users to be able to easily message their friends.

**Comment by akirk** [comment]
It just uses Gutenberg (on the frontend) to allow composing of rich messages. It works on multisite.

**Comment by Appshure** [comment]
![Screenshot_20220119-224041](https://user-images.githubusercontent.com/97206239/150220460-4ceebfe4-5c2d-48bd-94f1-ff82288fe878.jpg)

I finally figured it out, but I can't compose messages, the message body isn't clicking, only blinking.

**Comment by akirk** [comment]
Looks like https://github.com/Automattic/gutenberg-everywhere/ is currently broken. I have disabled it and released version 1.9.0. Could you check if it now works for you?

**Comment by Appshure** [comment]
It now works, thank you.
Another issue I found is that when I send a friend request to another blog, it says Friends plugin isn't installed on the site and shows this message:
 "The other side responded: An invalid codeword was provided.

No friends plugin installed at appshure. We subscribed you to their updates. Go to your friends page to view their posts."
When I click to "users" and click "resend friends request", the friends request sends.
But if the friends request in sent to the main site, it sends immediately.
